index.html

```diff
@@ -38,6 +39,156 @@
     </div>
 
   </section>
+
+  <section class="work-section">
+    <div class="RWbox"><div class="RWork"><p>My Recent works</p>
+    <div class="workline"></div></div>
+    </div>
+    <div class="work-container">
+      <div class="workone"><img class="workphoto"src="/img/Img Placeholder.svg" alt="Recent work preview">
+       <p class="first-Rwork">Multi-Post Stories</p>
+       <p class="work-dicreption">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <img class="languages-Fwork" src="/img/Tags.svg" alt="languages used work">
+        <img class="see-project" src="/img/Enable.svg" alt="see project botton">
+      </div>
+      <div class="diffrent-work">
+      <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
```

Comment:
- [ ] Kindly ensure that all your project buttons (e.g., "See project") are a link `a` or `button` tag, so that you will be able to see the main project when the button is clicked 😄